index.js

```diff
@@ -20,7 +28,15 @@ async function underPressure (fastify, opts) {
   var heapUsed = 0
   var rssBytes = 0
   var eventLoopDelay = 0
-  var lastCheck = now()
+  var h, lastCheck
+
+  if (monitorEventLoopDelay) {
+    h = monitorEventLoopDelay({ resolution })
+    h.enable()
+  } else {
+    lastCheck = now()
+  }
+
   const timer = setInterval(updateMemoryUsage, sampleInterval)
```

Comment:
This will also poll memory every 5ms. I think that's definitely too quickly (it creates too much overhead), it should really stay every second or so.

Reply:
I agree but it does that in the case that  `monitorEventLoopDelay` is not available, as it did before if I am not mistaken. Should I set it at one second anyway? It would simplify the getSampleInterval function.

Reply:
True, resolved